tests/unit_tests/gui/test_gui_load.py

```diff
@@ -1,443 +0,0 @@
-import argparse
-import os
-import shutil
-from pathlib import Path
-from unittest.mock import MagicMock, Mock, PropertyMock, patch
-
-import pytest
-from qtpy.QtCore import Qt, QTimer
-from qtpy.QtWidgets import QComboBox, QMessageBox, QToolButton, QWidget
-
-import ert.gui
-from ert.config import ErtConfig
-from ert.enkf_main import EnKFMain
-from ert.gui.about_dialog import AboutDialog
-from ert.gui.ertwidgets import SuggestorMessage
-from ert.gui.main import GUILogHandler, _setup_main_window, run_gui
-from ert.gui.simulation.run_dialog import RunDialog
-from ert.gui.tools.event_viewer import add_gui_log_handler
-from ert.gui.tools.plot.plot_window import PlotWindow
-from ert.services import StorageService
-from ert.shared.plugins.plugin_manager import ErtPluginManager
-
-from .conftest import get_child, wait_for_child
-
-
-@pytest.mark.usefixtures("use_tmpdir")
-def test_gui_load(qtbot):
-    config_file = Path("config.ert")
-    config_file.write_text("NUM_REALIZATIONS 1\n", encoding="utf-8")
-
-    args = Mock()
-    args.config = str(config_file)
-    gui = _setup_main_window(
-        EnKFMain(ErtConfig.from_file(str(config_file))), args, GUILogHandler()
-    )
-    qtbot.addWidget(gui)
-
-    sim_panel = get_child(gui, QWidget, name="Simulation_panel")
-    single_run_panel = get_child(gui, QWidget, name="Single_test_run_panel")
-    assert (
-        sim_panel.getCurrentSimulationModel() == single_run_panel.getSimulationModel()
-    )
-
-    sim_mode = get_child(gui, QWidget, name="Simulation_mode")
-    qtbot.keyClick(sim_mode, Qt.Key_Down)
-
-    ensemble_panel = get_child(gui, QWidget, name="Ensemble_experiment_panel")
-    assert sim_panel.getCurrentSimulationModel() == ensemble_panel.getSimulationModel()
-
-
-@pytest.mark.requires_window_manager
-def test_gui_full(monkeypatch, tmp_path, qapp, mock_start_server, source_root):
-    shutil.copytree(
-        os.path.join(source_root, "test-data", "poly_example"),
-        os.path.join(tmp_path, "poly_example"),
-    )
-
-    args = argparse.Namespace(config="poly_example/poly.ert", read_only=True)
-
-    monkeypatch.chdir(tmp_path)
-
-    qapp.exec_ = lambda: None  # exec_ starts the event loop, and will stall the test.
-    monkeypatch.setattr(ert.gui.main, "QApplication", Mock(return_value=qapp))
-    run_gui(args)
-    mock_start_server.assert_called_once_with(
-        project=str(tmp_path / "poly_example" / "storage"),
-        ert_config="poly.ert",
-    )
```

Comment:
You decided to remove these because they are not testing anything in particular other that starting the UI? I guess these tests have little to no value at the point where we are now.

Reply:
Yes, that is correct. I thought it made sense to leave them out as it is covered by the parametrized test.